ptable/ptable.h

```diff
@@ -0,0 +1,55 @@
+/*
+ * Phoenix-RTOS
+ *
+ * Partition table
+ *
+ * Copyright 2020 Phoenix Systems
+ * Author: Hubert Buczynski
+ *
+ * This file is part of Phoenix-RTOS.
+ *
+ * %LICENSE%
+ */
+
+
+#ifndef _LIB_PARTITION_TABLE_H_
+#define _LIB_PARTITION_TABLE_H_
+
+
+/* Structure of partition table:
+*  _________________________________________________________________________________
+* |        28 B     |                      32 B * n                     |    4 B    |
+*  ---------------------------------------------------------------------------------
+* | ptable_header_t | ptable_partition_t 0 | ... | ptable_partition_t n | magicBytes|
+*  ---------------------------------------------------------------------------------
+*/
+
+
+static const uint8_t magicBytes[] = {0xde, 0xad, 0xfc, 0xbe};
+
+
+/* Based on MBR types */
+enum { ptable_raw = 0x51, ptable_meterfs = 0x75 };
+
+
+typedef struct ptable_partition {
+	unsigned char name[8];
+
+	uint32_t accessMode;
+
+	uint32_t offset;
+	uint32_t size;
+
+	uint8_t type;
+	uint8_t reserved[11];     /* default value should be 0 */
+} __attribute__((packed)) ptable_partition_t;
+
+
+typedef struct ptable_header {
+	uint32_t pSize;           /* number of partitions */
```

Comment:
if `pSize` is _number of partitions_, it should be named `pCount` for example.

Or the comment should be changed to _byte size of the ptable_partition_t entries_ (or something similar).